cluster/manifests/infrastructure-configs/configmap.yaml

```diff
@@ -8,3 +8,4 @@ data:
   scalyr-team-token: "{{.Cluster.ConfigItems.scalyr_team_token}}"
   create-namespaces: "true"
   aws-available: "true"
```

Comment:
We can drop this or?

Reply:
Not yet, it's required for the current version. :(